### PR TITLE
Added inherited generic interfaces

### PR DESCRIPTION
**Pull request by yuri-voloshyn**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Added inherited generic interfaces

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**What might this PR break?**

**Please check if the PR fulfills these requirements**
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information**:

### CODE REVIEW

**Comment by yuri-voloshyn** [comment]
Fixed this issue https://github.com/reactiveui/refit/issues/595

File: InterfaceStubGenerator.Core/InterfaceStubGenerator.cs [hunk 1]
```diff
@@ -252,26 +365,73 @@ public class ClassTemplateInfo
...
+        }
+
+        public TypeInfo Clone()
+        {
+            return CloneImpl() as TypeInfo;
+        }
+
+        object ICloneable.Clone()
+        {
+            throw new NotImplementedException();
```
Comment (onovotny) [inline InterfaceStubGenerator.Core/InterfaceStubGenerator.cs]: Shouldn't this return `Clone()`?

**Comment by gabolarraguivel** [comment]
Hi! I dont know where to ask this but I have a problem that this commit solves. How can I get or build a version with this commit in in?
Thank you

**Comment by yuri-voloshyn** [comment]
Hi!
You can download my fork https://github.com/yuri-voloshyn/refit or add
comment to pull request that should be accepted.

пт, 12 апр. 2019 г. в 04:43, gabolarraguivel <notifications@github.com>:

**Comment by onovotny** [comment]
This should be in the latest published version.

**Comment by gabolarraguivel** [comment]
Thank you very much!

**Pull request merged**